python/tvm/relay/frontend/pytorch.py

```diff
@@ -213,12 +213,33 @@ def _impl(inputs, input_types):
         pool_size = _infer_shape(inputs[1])
         strides = _infer_shape(inputs[2])
         padding = _infer_shape(inputs[3])
-
+        dilation = _infer_shape(inputs[4])
```

Comment:
does pooling has dilation argument?

Reply:
In https://pytorch.org/docs/stable/nn.html#maxpool1d, MaxPool1d and MaxPool2d have dilation argument.